app/controllers/v0/form1095_bs_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module V0
+  class Form1095BsController < ApplicationController
+    before_action { authorize :form1095, :access? }
+    before_action :get_1095b, only: :download
+    before_action :get_available_forms, only: :available_forms
```

Comment:
Could you use the rails standard naming convention here? `before_action :set_<instance_variable_name>`
```suggestion
    before_action :set_form, only: :download
    before_action :set_available_forms, only: :available_forms
```